client/templates/shared/course-card.html

```diff
@@ -7,7 +7,7 @@
                 </a>
             </div>
             <div class="caption">
-                <a class="course-category" href="#">Mathematics</a>
+              <a class="course-category" href="/learn/{{keywords.[0]}}">{{keywords.[0]}}</a>  
```

Comment:
The course category and keyword tags are intended as separate fields.  For the Course Category feature, you will need to add a new field to the `teach` template, called `category`. The `category` field will offer a pre-defined list of `allowedValues`, such as:

* Science & Technology
* Humanities & Art
* Medicine and Health
* Social Sciences
* Other

Please see the ["Course subjects or topics" design document](https://crowducate.hackpad.com/Course-subjects-or-topics-Z3E7jEYxu1E) for further details.